docs/changelog.md

```diff
@@ -4,9 +4,18 @@ title: Changelog
 sidebar_label: Changelog
 ---
 
+## v2.5. - Dec 7, 2022
+
+### Bug Fixes
+
+- Fixed bugs in the C++ API concerning constructors and assignment operators.
+- Fixed a bug in the C++ API’s `Value::ValueNumeric` method.
+- Added `SetErrorMessage` methods to the C++ API for uniformity with the C API.
+
```

Comment:
can you please make these notes more user oriented... yes we fixed it, but what does that mean for the user... what can he do now, what will work that didn't before